R/utils.r

```diff
@@ -69,7 +69,7 @@ rsync <- function(from, to, args = "-rtlzv --delete") {
 query <- function(url, error = TRUE, ...) {
 
   result <- url %>%
-    httr::GET() %>%
+    httr::GET(...) %>%
     content(as = "text", encoding = "UTF-8") %>%
     fromJSON(...)
 
```

Comment:
Actually, padding `...` _both_ to `GET()` and `fromJSON()` seems quite silly. This is probably never used, otherwise it would cause an error. I guess we could remove the `...` from `fromJSON()`, but you don't have to do it in this PR.